Dockerfile

```diff
@@ -64,3 +64,43 @@ COPY --from=secrets-provider-builder /opt/secrets-provider-for-k8s/secrets-provi
 
 ENTRYPOINT [ "/usr/local/bin/secrets-provider"]
 
+# =================== MAIN CONTAINER (REDHAT) ===================
+FROM registry.access.redhat.com/rhel as secrets-provider-for-k8s-redhat
+MAINTAINER CyberArk Software, Ltd.
+
+ARG VERSION
+
+LABEL name="secrets-provider-for-k8s"
+LABEL vendor="CyberArk"
+LABEL version="$VERSION"
+LABEL release="$VERSION"
+LABEL summary="Store secrets in Conjur or DAP and consume them in your Openshift application containers"
```

Comment:
I would add cyberark and maybe link to the docs?

Reply:
which docs, oss or dap?

Reply:
let's not add DAP docs in an open-source project.

Reply:
Let's solidify terminology and the type experience we want to offer to our customers. The github repo is named `Secrets Provider for K8s` but after a conversation w/ @InbalZilberman yesterday, we are naming the RH project `Secrets Provider for Openshift`. It is important to note that we also see traces of this shift in terminology for our authn client `Openshift Authenticator Client` although in our documentation we reference it as `Kubernetes Authenticator Client`.

Is this the direction we also want to go in? If so, should we remove traces of "Kubernetes" in our RH descriptions. For example, in the description, should it be:

```
The Secrets Provider for Openshift (for Kubernetes / for Kubernetes and Openshift) runs as an init container and fetches
```
I would like to get opinions here... @InbalZilberman @Tovli @orenbm

Reply:
i don't have a strong opinion on this. I would like to get @izgeri 's opinion on this as she handled RH for the authn-client.

I'll go with whatever is decided in this thread.